nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -214,6 +215,22 @@
             .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor PROP_IDLE_TIMEOUT = new PropertyDescriptor.Builder()
+            .name("Idle Timeout")
+            .description("Max idle time before closing connection to the remote service.")
+            .required(true)
+            .defaultValue("5 mins")
+            .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
```

Comment:
I think this needs to validate that it's > 0. OkHttp appears to, but it'd probably be nice if it got caught beforehand; see https://github.com/square/okhttp/blob/cfdeb570c211f83cef23c00afb18fe87c1a292f2/okhttp/src/main/kotlin/okhttp3/internal/connection/RealConnectionPool.kt#L49-L52

You can use `StandardValidators.createTimePeriodValidator()` with a minimum of 1 sec or something.